examples/server_profiles.py

```diff
@@ -169,6 +169,22 @@
 except HPOneViewException as e:
     print(e.msg)
 
+try:
+    # Get Server Profile by scope_uris
```

Comment:
This comment is wrong. Please refer REST API docs to know what this call does. 

`The scopeUris, serverHardwareTypeUri, serverHardwareUri and enclosureGroupUri parameters should be specified to get the available networks for a new server profile`